module/x/gravity/keeper/ethereum_event_vote.go

```diff
@@ -244,30 +256,9 @@ func (k Keeper) getLastEventNonceByValidator(ctx sdk.Context, validator sdk.ValA
 		// them to replay the entire history of all events ever we can't start
 		// at zero
 		//
-		// We could start at the LastObservedEventNonce but if we do that this
```

Comment:
Removed this section.

May be some old code?

The validator arent actually getting slashed for not submitting ethereum events and we are not pruning the event neither